### PR TITLE
Simplify install, rearrange dir structure, better dependency management

### PR DESCRIPTION
**Pull request by chao-mu**
In addition to the changes in the title, now generates a combined TSV.

### CODE REVIEW

File: bin/gen.py [hunk 1]
```diff
@@ -45,16 +45,17 @@ def warning(self, lno: int, err_msg: str) -> None:
...
+   
+    expected_header = ["eco", "name", "pgn"]
     for lno, line in enumerate(f, 1):
-        cols = line.rstrip("\n").split("\t")
+        cols = line.rstrip("\n").split("\t")[:3]
 
-        if len(cols) != 3:
-            reporter.error(lno, f"expected 3 columns, got {len(cols)}")
+        if len(cols) < 3:
+            reporter.error(lno, f"expected at least 3 columns, got {len(cols)}")
```
Comment (niklasf) [inline bin/gen.py:54]: What's the idea here, i.e., why allow additional colummns?

Comment (chao-mu) [reply bin/gen.py:54]: I believe this is for b when I collapse all generated into all.tsv

Comment (niklasf) [reply bin/gen.py:54]: Sorry, I don't understand. Can you please explain a bit more?

Comment (chao-mu) [reply bin/gen.py:54]: Take a peek at the Makefile. I'm now building an all.tsv that combines the others. Though now I'm wondering if this particular change is for code that didn't make it in....

**Comment by niklasf** [comment]
One more question: Is the move to `data/` necessary to help with some kind of tooling, or just cosmetics?

**Comment by chao-mu** [comment]
Cosmetics and ergonomics,if I recall.

File: bin/gen.py (hunk 1 above)
Comment (chao-mu) [reply bin/gen.py:54]: Are there any changes I should make?

**Comment by niklasf** [comment]
Thanks! I have adopted some parts, but also rejected some - sorry about that. Rationale:

* .gitignore additions: Imo public projects should not have IDE-specific ignores.
* dist/all.tsv: Looks very useful. Creating this without relaxing the number of columns in source files.
* Moving source files: Avoiding this for now, to not cause unnescessary churn for downstream projects.
* Unsure about pyproject.toml, and if adopting it would imply that this data set is itself an installable python library.

**Pull request closed without merging**